infra/manifests/billing-service-secondary-async.dev.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: microservices
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: billing-service-secondary-async
+  namespace: microservices
+spec:
+  selector:
+    app: billing-service-secondary-async
+  ports:
+    - name: http
```

Comment:
port 8080 is not needed on service, there is no http vs https in-cluster. it's all http, ssl termination is done on ingress.

Reply:
fixed